lib/util/getfstype.c

```diff
@@ -62,7 +62,7 @@
 struct ftyperec {
 	unsigned int f_type;
 	const char * typename;
-} ftypes[] = {
+} static ftypes[] = {
```

Comment:
Normally we put `static` before the type... is there some reason to do otherwise here?